### ISSUE TITLE
 Illegal invocation w/ 2.5.0

### ISSUE THREAD

**Issue by chrisnojima** [facebook/react-devtools#849]
I'm debugging an electron app and when I click on HTMLElements on the right side pane I get console.errors

Uncaught TypeError: Illegal invocation
    at chrome-extension://react-developer-tools/build/backend.js:7482:54
    at Array.forEach (native)
    at Bridge._inspectResponse (chrome-extension://react-developer-tools/build/backend.js:7478:54)
    at Bridge._handleMessage (chrome-extension://react-developer-tools/build/backend.js:7360:15)
    at listener (chrome-extension://react-developer-tools/build/backend.js:86:10)

which points to

```
if (!protod && val.__proto__ && val.constructor.name !== 'Object') {
	          var newProto = {};
	          var pIsFn = typeof val.__proto__ === 'function';
	          Object.getOwnPropertyNames(val.__proto__).forEach(function (name) {
	            if (pIsFn && (name === 'arguments' || name === 'callee' || name === 'caller')) {
	              return;
	            }
>>>>>	            newProto[name] = dehydrate(val.__proto__[name], protoclean, [name]);
	          });
	          proto = newProto;
	        }
```

**Comment by bvaughn** [comment]
Sorry for the silly question, but could you elaborate on what you mean by "_clicking on HTMLElements on the right side pane_"?

Do you think there's something special about your Electron app WRT this bug or do you think it's reproducible with any app?

I cloned [chentsulin/electron-react-boilerplate](https://github.com/chentsulin/electron-react-boilerplate), upgraded the version of devtools it uses to 2.5.0, and tried clicking around a bit and was unable to reproduce anything like what you mentioned. Seems to work fine for me.

**Comment by chrisnojima** [comment]
Sorry for the ambiguity! By clicking on an HTMLElement I mean, I had a react node with a prop that was a result from a ref (so a dom node). When I click on it to I would get this error. I tried to make a simple repro using the electron-react-boilerplate but it seemed to work fine.

The PR where I noticed this is kinda strange so it might be a confluence of a lot of weird things (Using react 16 alpha, etc). I say we close it for now. thanks!

**Comment by bvaughn** [comment]
If you're able to provide a minimal repro of this issue, please tag me here. I'd like to take a look. 😄

**Comment by chrisnojima** [comment]
Hi,

I think this is related to making calls w/o the correct context. In my app I had a console.log override which I think is the real culprit (basically this is happening: https://bugs.chromium.org/p/chromium/issues/detail?id=167911&q=illegal%20invocation&colspec=ID%20Pri%20M%20Week%20ReleaseBlock%20Cr%20Status%20Owner%20Summary%20OS%20Modified)

So when redux-devtools was using console.log it was causing this to happen. 

So tldr; let's keep this closed!

Thanks
